azuread/table_azuread_admin_consent_request_policy.go

```diff
@@ -0,0 +1,60 @@
+package azuread
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureAdAdminConsentRequestPolicy(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azuread_admin_consent_request_policy",
+		Description: "Represents the policy for enabling or disabling the Azure AD admin consent workflow.",
+		List: &plugin.ListConfig{
+			Hydrate: listAdAdminConsentRequestPolicies,
+		},
+
+		Columns: []*plugin.Column{
+			{Name: "id", Type: proto.ColumnType_STRING, Description: "", Default: "adminConsentRequestPolicy"},
+			{Name: "is_enabled", Type: proto.ColumnType_BOOL, Description: "Specifies whether the admin consent request feature is enabled or disabled.", Transform: transform.FromMethod("GetIsEnabled")},
+
+			// Other fields
+			{Name: "notify_reviewers", Type: proto.ColumnType_BOOL, Description: "Specifies whether reviewers will receive notifications.", Transform: transform.FromMethod("GetNotifyReviewers")},
+			{Name: "reminders_enabled", Type: proto.ColumnType_BOOL, Description: "Specifies whether reviewers will receive reminder emails.", Transform: transform.FromMethod("GetRemindersEnabled")},
+			{Name: "request_duration_in_days", Type: proto.ColumnType_INT, Description: "Specifies the duration the request is active before it automatically expires if no decision is applied.", Transform: transform.FromMethod("GetRequestDurationInDays")},
+			{Name: "version", Type: proto.ColumnType_INT, Description: "Specifies the version of this policy. When the policy is updated, this version is updated.", Transform: transform.FromMethod("GetVersion")},
```

Comment:
Is this always an int, and not a string? For instance, does it have any characters like `v`, or does it have a format like `1.0.0`?

Reply:
@cbruno10 As per the API docs, `version` is of type `Int32`.
https://learn.microsoft.com/en-us/graph/api/resources/adminconsentrequestpolicy?view=graph-rest-1.0#properties

Here is the sample response:
https://learn.microsoft.com/en-us/graph/api/resources/adminconsentrequestpolicy?view=graph-rest-1.0#properties